### PR TITLE
Add login route based on isMfaEnabled

### PR DESCRIPTION
**Pull request by ddrosario**

### CODE REVIEW

File: server/routes/auth/local.js [hunk 1]
```diff
@@ -37,9 +37,15 @@ router.post('/login', (req, res, next) => {
         res.status(HttpStatus.INTERNAL_SERVER_ERROR).end();
       }
     } else if (user) {
-      req.login(user, () => {
+      req.login(user, async () => {
         if (req.accepts('html')) {
-          res.redirect('/auth/local/twoFactor');
+          const org = await user.getOrganization();
+          if (org.toJSON().isMfaEnabled) {
```
Comment (francisli) [inline server/routes/auth/local.js:43]: You don't need to convert the object instance to JSON to access the attribute...

**Comment by francisli** [comment]
@ddrosario looks like the CI check failed on code formatting... you can run `npm run format` inside the running container to run the prettier formatter 

I also suspect all the tests will need to be modified, as they are currently expecting two factor to be on, but the test organization fixures will be default to OFF.

We'll probably want to have separate explicit tests for testing logging in with/without MFA enabled.

**Comment by francisli** [comment]
You can run just the server tests inside the running container with `npm test -w server`

File: server/routes/auth/local.js (hunk 1 above)
Comment (ddrosario) [reply server/routes/auth/local.js:43]: I forgot to restart the docker containers when checking out the branches. I think I had that up when I was originally trying to debug why it was missing. Will fix

**Comment by ddrosario** [comment]
This is the error I am getting in the container. `SequelizeConnectionError: database "bats_test" does not exist`
I'll look into it

**Comment by francisli** [comment]
> This is the error I am getting in the container. `SequelizeConnectionError: database "bats_test" does not exist` I'll look into it

The startup scripts only check/create the main app database, but a separate database is used for tests (since tests create/erase data). To create it, you can run this in the container in the server directory: `sequelize db:create --env=test`

**Comment by francisli** [comment]
@ddrosario do you have time to continue working on this branch and getting tests to pass?

**Comment by ddrosario** [comment]
> @ddrosario do you have time to continue working on this branch and getting tests to pass?

Yes I do. Sorry this last weekend was really busy but I'll be able to work on it

**Comment by ddrosario** [comment]
Also it looks like the server tests are fine. After running `npm run test` the last 2 tests in the e2e tests are the ones failing.

**Comment by francisli** [comment]
> Also it looks like the server tests are fine. After running `npm run test` the last 2 tests in the e2e tests are the ones failing.

Ok, great, no worries on being busy...!

**Review by francisli — APPROVED** [review]

**Pull request merged**